src/Build/Definition/ProjectImportPathMatch.cs

```diff
@@ -24,36 +32,39 @@ internal ProjectImportPathMatch(string propertyName, List<string> searchPaths)
             ErrorUtilities.VerifyThrowArgumentNull(propertyName, nameof(propertyName));
             ErrorUtilities.VerifyThrowArgumentNull(searchPaths, nameof(searchPaths));
 
-            PropertyName = propertyName;
-            SearchPaths = searchPaths;
-            MsBuildPropertyFormat = $"$({PropertyName})";
+            _propertyName = propertyName;
+            _searchPaths = searchPaths;
+            _msBuildPropertyFormat = $"$({PropertyName})";
         }
 
         public ProjectImportPathMatch(ITranslator translator)
         {
             ((ITranslatable)this).Translate(translator);
         }
 
+        internal ProjectImportPathMatch DeepClone()
```

Comment:
If it's immutable, does it need to be cloned?

Reply:
ah good point - removing...

Reply:
(removed)